src/components/projecteditor/control.js

```diff
@@ -697,9 +697,11 @@ export default class DevkitProjectEditorControl extends Component {
             if(status==0) {
                 // TODO: this lookup is bad.
                 const ctrs=projectItem.props.state.children[0].props.state._children;
-                const contract=ctrs[ctrs.length-2];
-                const item=contract.props.state.children[0];
-                if(this.props.router.panes) this.props.router.panes.openItem(item);
+                if(ctrs) {
```

Comment:
I would check also that the length is >=2 otherwise would not make sense and a new exeception will be raised. Why ctrs[ctrs.lenght-2] though?